cmd/web3/main.go

```diff
@@ -255,6 +260,100 @@ func main() {
 							Hidden:      false},
 					},
 				},
+				{
+					Name:  "upgrade",
+					Usage: "Upgrade contract to new address",
+					Action: func(c *cli.Context) {
+						UpgradeContract(ctx, network.URL, privateKey, contractAddress, toContractAddress, amount)
+					},
+					Flags: []cli.Flag{
+						cli.StringFlag{
+							Name:        "address",
+							Destination: &contractAddress,
+							Usage:       "Proxy contract address",
+							Hidden:      false},
+						cli.StringFlag{
+							Name:        "to",
+							Destination: &toContractAddress,
+							Usage:       "Contract address to upgrade to",
+							Hidden:      false},
+						cli.IntFlag{
+							Name:        "amount",
+							Destination: &amount,
+							Usage:       "Amount in wei that you want to send to the transaction",
+							Hidden:      false},
+						cli.StringFlag{
+							Name:        "private-key",
+							Usage:       "Private key",
+							EnvVar:      "WEB3_PRIVATE_KEY",
+							Destination: &privateKey,
+							Hidden:      false},
+					},
+				},
+				{
+					Name:  "target",
+					Usage: "Return target address of upgradeable proxy",
+					Action: func(c *cli.Context) {
+						GetTargetContract(ctx, network.URL, contractAddress)
+					},
+					Flags: []cli.Flag{
+						cli.StringFlag{
+							Name:        "address",
+							Destination: &contractAddress,
+							Usage:       "Proxy contract address",
+							Hidden:      false},
+					},
+				},
+				{
+					Name:  "pause",
+					Usage: "Pause an upgradeable contract",
+					Action: func(c *cli.Context) {
+						PauseContract(ctx, network.URL, privateKey, contractAddress, amount)
+					},
+					Flags: []cli.Flag{
+						cli.StringFlag{
+							Name:        "address",
```

Comment:
For `pause` and `resume`, would it make sense to look for the address as the first argument, rather than a flag?
```web3 contract pause 0x1234```
That would be in line with some of the other commands.